pkg/rpc/context.go

```diff
@@ -107,52 +108,118 @@ type connMeta struct {
 	heartbeatErr error
 }
 
-// Context contains the fields required by the rpc framework.
-type Context struct {
+// ContextTestingKnobs are the testing knobs for a Context.
+type ContextTestingKnobs struct {
+	BreakerFactory func() *circuit.Breaker
+	// PeriodicClockCheckHook is used to override the default clock offset check
+	// on connections. If an error is returned, the node will crash itself.
+	PeriodicClockCheckHook func() error
+}
+
+// ModuleTestingKnobs is part of the base.ModuleTestingKnobs interface.
+func (*ContextTestingKnobs) ModuleTestingKnobs() {
+}
+
+var _ base.ModuleTestingKnobs = &ContextTestingKnobs{}
+
+// ContextConfig contains parameters for creating a Context.
+type ContextConfig struct {
```

Comment:
Given the confusion we have between the terms "context" and "config", a type like `ContextConfig` makes my head spin :) Can it just be `rpc.Config`?